pages/encrypt/social.jsx

```diff
@@ -93,19 +116,32 @@ module.exports = React.createClass({
         <Footer>
           <Icon href="https://medium.com/encryption-matters" src="/assets/footer-icon-medium.svg" title="Medium">Join the Conversation</Icon>
         </Footer>
-        <div hidden={!this.state.formIsVisible}>
-          <Modal className="join-modal" hideModal={this.hideModal}>
-            <div className="cta">
-              <h2 aria-role="label">Join the email list</h2>
-              <div className="horizontal-rule"></div>
-              <Signup formName="signup" submitButtonText="Subscribe" onSubmission={this.userDidSignup}/>
+        <div hidden={!this.state.formIsVisible && !this.state.videoDidEnd}>
+          <Modal hideModal={this.hideModal} className="pledge-cta">
+            <div>
+              <p className="pledge-cta-title">Pledge to stand up for strong encryption</p>
+              <p className="pledge-cta-info">We’re seeing more and more governments attempt to undermine encryption.Will you help others understand what encryption is and stand up for strong encryption when it matters most?Sign on to be an encryption champion.</p>
+            </div>
+            <div>
+              <p className="pledge-cta-why">Let us know what country you’re in so if debates are happening in your country we can email you the most relevant updates.We’ll also send you useful tips about how to use encryption that you can share with friends.</p>
+              <Signup dataToPrefill={this.knownUserInfo} onSubmission={this.userDidSignup} formName="afterVideo" />
             </div>
           </Modal>
         </div>
-        <div hidden={!this.state.videoDidEnd}>
-          <Modal hideModal={this.hideModal} className="postVideo signup-cta">
-            <p className="cta-title">Stand with us for a free and open Internet.<br/>Sign on.</p>
-            <Signup formName="afterVideo" />
+        <div hidden={!this.state.shareModalIsVisible}>
+          <Modal hideModal={this.hideModal} className="postVideo social-cta">
```

Comment:
totally a nitpick - would be extra awesome if we can turn `postVideo` into `post-video` to follow the convention. This can totally be fixed in a followup PR though.